app/jobs/compact_uploads_job.rb

```diff
@@ -5,6 +5,22 @@
 class CompactUploadsJob < ApplicationJob
   queue_as :default
 
+  def self.enqueue_some(ratio: 0.25, maximum: 5) # rubocop:disable Metrics/AbcSize
+    streams = Stream.include(:statistic).default.select do |stream|
```

Comment:
I think this should be `includes`?
```suggestion
    streams = Stream.includes(:statistic).default.select do |stream|
```